seqr/management/tests/run_postgres_database_backup_2_3_tests.py

```diff
@@ -1,24 +1,24 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
 import mock
-from io import BytesIO
 import datetime
 
 from django.core.management import call_command
 from django.test import TestCase
 
 
+@mock.patch('seqr.management.commands.run_postgres_database_backup.logger')
 @mock.patch('seqr.management.commands.run_postgres_database_backup.os')
 @mock.patch('seqr.management.commands.run_postgres_database_backup.datetime')
 class RunProtgresDatabaseBackupTest(TestCase):
 
     # Test the command without an argument
-    def test_command_no_argument(self, mock_datetime, mock_os):
-        out = BytesIO()
-
+    def test_command_no_argument(self, mock_datetime, mock_os, mock_logger):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5795824)

Reply:
not relevant